Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -504,7 +504,7 @@ std::unique_ptr<MaybeDocument> Serializer::DecodeBatchGetDocumentsResponse(
   // Initialize BatchGetDocumentsResponse fields to their default values
   std::unique_ptr<MaybeDocument> found;
   std::string missing;
-  // TODO(rsgowman): transaction
+  // We explicitly ignore the 'transaction' field
```

Comment:
Nit: consider maybe moving this comment down to the switch, so that lines 527-530 become:
```cpp
case ...transaction_tag:
  // We explicitly ignore...
  break;
```
Feel free to leave as is, too.

Reply:
I'm going to leave it where it is for now, because:
a) there's two locations where we look to see if the tag is "txn", both here and below
b) I don't want to simply break, i.e. I still want to ensure the type is correct, and to skip those bytes. (Technically, we don't need to enforce type correctness here, but we do for other known fields, so I figure we should do so for known ignored fields too. This could potentially be relaxed in the future.)